homepage/homepage/app/page.mdx

```diff
@@ -268,11 +268,7 @@ Jazz Mesh is currently free &mdash; and it's set up as the default sync & storag
 
 ## Get Started
 
--   <Link href="/docs" target="_blank">
-        Read the docs
-    </Link>
```

Comment:
The newline here drives the MDX processor to add a `p` tag around the text breaking the layout.

<img width="410" alt="Screenshot 2024-08-10 at 16 20 35" src="https://github.com/user-attachments/assets/755c9ad4-52cb-49f8-8f3b-42f6de55019b">

The problem is that Prettier automatically inserts this newline so it's very easy to introduce the bug again.

**suggestion:** would be better to disable the Prettier formatting on MDX files as it is easy to incur in this kind of problems

Reply:
Good point! https://github.com/gardencmp/jazz/issues/303